Misc/NEWS.d/next/Core and Builtins/2022-05-20-13-32-24.gh-issue-15870.e9B-pv.rst

```diff
@@ -0,0 +1,13 @@
+Added the new function :c:func:`PyType_FromMetaModuleAndSpec`, which mirrors
+the behavior of :c:func:`PyType_FromModuleAndSpec`, except that it takes an
+additional metaclass argument. This is useful for language binding tools, where
+it enables uses casies including
```

Comment:
Small typo:
```suggestion
it enables use cases including
```

Reply:
good catch, I amended the commit. 👍

Reply:
Thx! Still an s too much in "use~~s~~ cases" though :)

Reply:
oops 🤦